### PR TITLE
Test Backend Auth & Organizations

### PR DESCRIPTION
**Pull request by Melisa96**
## Description

* REGISTRO
    * When user credentials are valid
        - Returned userId belongs to an existing user in the db
        - user create should respond with a 201 status code
    * When user credentials are invalid
        - when password is null, responds with 401 status code
        - when email is not email format , responds with 400 status code
* LOGIN
    - when password is correct, responds with unauthorized false
    - when password is incorrect, responds with unauthorized true
    - when email is not registered, responds with 401 status code
  

ORGANIZACIONES

* POST GET /api/organizations/
    * When user is not authorized - sin log-in
        * - should return a 401 status code
        * - should'n have permission to create organization, expect unauthorized true
    * When user is authorized  - token
        * - Should be able to create organization, expect 201 status code
        * - User should be able to see her organizations created GET /api/organizations/ expect organizationData toMatchObject from response
        * - User should be able to see her organizations by id GET  api/organizations/:organizationId  expect organizationData toMatchObject from response
		
    * When involves another user authorization
        * - Should not see the organization created from another user without permission
        * - Invite user to organization - Should see the organization created from another user when is invited to participate in.

## Requirements

Test Jest
Supertest

## Additional changes

Otro cambio que hice fue en el archivo src/index.js
se agregaron las líneas 
import orm_config from "./config/db.js"
orm_config()

### CODE REVIEW

File: src/routes/organization.routes.js [hunk 1]
```diff
@@ -20,16 +21,22 @@ router.get(
...
-  createProjectController
+  addUserToOrganizationController
 )
 
+// router.post(
+//   "/:organizationId/project",
+//   [verifyToken, verifyOrganizationPermission],
+//   createProjectController
+// )
+
```
Comment (FeBalla) [inline src/routes/organization.routes.js:41]: Esto debería borrarse?

File: src/routes/project.routes.js [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+
+router.get("/:projectId", [verifyToken, verifyReadProjectPermission], getProjectByIdController)
+
+router.delete("/:projectId", [verifyToken, verifyReadProjectPermission], deleteProjectController)
+
+// router.post(
+//   "/:projectId/user",
+//   [verifyToken, verifyReadProjectPermission],
+//   addUserToProjectController
+// )
```
Comment (FeBalla) [inline src/routes/project.routes.js:25]: Esto debería borrarse?

File: src/config/db.js [hunk 3]
```diff
@@ -32,20 +24,24 @@ export const Project = ProjectModel(sequelize, Sequelize)
 export const ProjectPermission = ProjectPermissionModel(sequelize, Sequelize)
 export const ProjectImage = ProjectImageModel(sequelize, Sequelize)
 
-//Associations
-User.belongsToMany(Organization, { through: "OrganizationPermission", as: "userOrganizations" })
-Organization.belongsToMany(User, { through: "OrganizationPermission", as: "organizationUsers" })
+const orm_config = async () => {
```
Comment (FeBalla) [inline src/config/db.js:27]: cambiaría el nombre por camel case, tipo  `ormConfig`

**Review by NicoMeyer — APPROVED** [review]
Functional App and Testing. Still remains making the testing faster.

**Review by ignaciomansuy — APPROVED** [review]
looking, must check if tests included in CI

**Pull request merged**